### PR TITLE
Absences and reservations features 

### PR DESCRIPTION
**Pull request by Igifigi**
This PR brings some new features and fixes bugs by removing the display of reservations. Main features:
- form for adding absences (#1),
- ability to delete reservations and absences from the absence management view, as like with substitutions,
- fix the display of absences,
- removed the display of reservations in room timetable and rooms view due to poor and bug-prone implementation.

### CODE REVIEW

**Comment by opti0** [comment]
Closes #1

File: locale/pl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -273,30 +304,35 @@ msgstr ""
...
 
+#: timetable/templates/add_absence.html:5
+#| msgid "Absences"
+msgid "Group absences"
+msgstr "Nieobecności grup"
+
+#: timetable/templates/add_absence.html:10
+#| msgid "Adding reservations"
+msgid "Adding group absences"
+msgstr "Dodawanie rezerwacji"
```
Comment (opti0) [inline locale/pl/LC_MESSAGES/django.po:333]: `msgstr "Dodawanie nieobecności grup"`

File: locale/pl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -260,11 +258,44 @@ msgid "CSV file with substitutions"
...
-#: timetable/templates/room_timetable.html:5
-#: timetable/templates/rooms.html:18
+#: timetable/templates/room_timetable.html:5 timetable/templates/rooms.html:18
 msgid "Room"
 msgstr "Sala"
 
+#: timetable/forms.py:175
+#| msgid "Select date and period"
+msgid "Start period"
+msgstr "Okres początkowa"
```
Comment (opti0) [inline locale/pl/LC_MESSAGES/django.po:268]: "Początek nieobecności" albo "Lekcja początkowa"

File: locale/pl/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -260,11 +258,44 @@ msgid "CSV file with substitutions"
...
+
+#: timetable/forms.py:176
+#| msgctxt "timetable header"
+#| msgid "Period"
+msgid "End period"
+msgstr "Okres końcowy"
```
Comment (opti0) [inline locale/pl/LC_MESSAGES/django.po:274]: "Koniec nieobecności" albo "Lekcja końcowa"

File: locale/pl/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -260,11 +258,44 @@ msgid "CSV file with substitutions"
...
+#: timetable/forms.py:179 timetable/templates/absences.html:12
+#: timetable/templates/group_timetable.html:7
+#: timetable/templates/group_timetable.html:18
+#: timetable/templates/rooms.html:21 timetable/templates/substitutions.html:22
+msgid "Group"
+msgstr "Grupa"
+
+#: timetable/forms.py:191
+msgid "Provide initial and final period."
+msgstr "Proszę wprowadzić poprawny początkowy i końcowy okres."
```
Comment (opti0) [inline locale/pl/LC_MESSAGES/django.po:293]: Wprowadź poprawną lekcję rozpoczęcia i zakończenia nieobecności.

**Review by opti0 — CHANGES_REQUESTED** [review]
Some changes in translations required.

**Comment by Igifigi** [comment]
Everything fixed.

**Review by opti0 — APPROVED** [review]

**Pull request merged**